.github/workflows/build_deploy_exm_fe.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and Deploy EXM FE
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+
+    - name: Install dependencies
+      run: npm install
+
+    - name: Build Angular application
+      run: npm run build --prod
+
+    - name: Terraform Output - Get VM IP
+      id: get-vm-ip
+      run: echo "::set-output name=vm_ip::$(terraform output -raw public_ip)"
```

Comment:
Nu cred ca o sa mearga cu terraform output aici, in primul rand nu rulezi terraform in pipeline-ul asta si in al doilea rand terraformul e in alt repo, deci runnerul care va rula pipeline-ul nici nu o sa aiba access la terraformul nostru. Ai putea sa incerci o comanda de azure cli care sa ia ip-ul de la vm. Va trebui facut Azure login si aici ca sa mearga cu az cli